test/runtests.jl

```diff
@@ -61,4 +61,12 @@ if !Sys.isfreebsd() # GR_jll not available, so Plots won't install
     profile = data_profile(PGFPlotsXBackend(), H, T, labels)
     @test isa(profile, Plots.Plot)
   end
+
+  @testset "csv export" begin
+    T = 10 * rand(25, 3)
+    filename = "data.csv"
+    export_performance_profile(T,filename)
```

Comment:
For the sake of keeping a good code coverage could you also add a test where the user provide another header?